lib/msal-native-auth/src/auth_flow/result/GetAccessTokenResult.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AuthenticationResult } from "@azure/msal-browser";
+import { ResultBase } from "./ResultBase.js";
```

Comment:
Use the `ts` extension since these are ts files

Reply:
This is not possible since we are using ESM modules in this project. This is the convention of all projects in this repo.